open_tms_backend/open_tms_create_department/index.js

```diff
@@ -2,18 +2,79 @@ import { shared } from '@appblocks/node-sdk'
 
 const handler = async (event) => {
```

Comment:
Please use examples/list_tickets as reference and add swagger documentation here to the handler so that this can be tested via swagger.Also refer to contribution.md.